app/Controller/FeedsController.php

```diff
@@ -488,26 +488,30 @@ public function fetchFromFeed($feedId)
             }
         }
         if (Configure::read('MISP.background_jobs')) {
-            $this->loadModel('Job');
-            $this->Job->create();
-            $data = array(
-                'worker' => 'default',
-                'job_type' => 'fetch_feeds',
-                'job_input' => 'Feed: ' . $feedId,
-                'status' => 0,
-                'retries' => 0,
-                'org' => $this->Auth->user('Organisation')['name'],
-                'message' => __('Starting fetch from Feed.'),
+
+            /** @var Job $job */
+            $job = ClassRegistry::init('Job');
+            $jobId = $job->createJob(
+                $this->Auth->user(),
+                Job::WORKER_DEFAULT,
+                'fetch_feeds',
+                'Feed: ' . $feedId,
+                __('Starting fetch from Feed.')
             );
-            $this->Job->save($data);
-            $jobId = $this->Job->id;
-            $process_id = CakeResque::enqueue(
-                'default',
-                'ServerShell',
-                array('fetchFeed', $this->Auth->user('id'), $feedId, $jobId),
-                true
+
+            $this->Feed->getBackgroundJobsTool()->enqueue(
```

Comment:
This doesn't work for the legacy background processor.

Reply:
My bad, the issue was an upstream change from MISP:develop that broke background processes.